pages/community/support-enquiry.html

```diff
@@ -0,0 +1,68 @@
+title: Support Enquiry
+description: This document gives you instructions how to provide relevant information for support enquiries when reporting issues with your Crate installation.
+
+{% extends "base.html" %}
+
+{% block body %}
+<div class="wrapper wrapper-bg-primary">
+  <div class="container">
+    <h1>Reporting Issues</h1>
+    <p class="lead">{{ description }}</p>
+  </div>
+</div>
+
+<div class="wrapper wrapper-border">
+  <div class="container">
+      <p>If you have general questions about Crate, e.g. how to use certain Crate specific data types or how to perform fulltext search, etc. just go ahead and drop us a email at <a href="mailto:support@crate.io"><b>support@crate.io</b></a>, join our <a href="https://gitter.com/crate/crate">Gitter chat</a> or chat using the <a href="https://intercom.io">Intercom widget</a> on our website.</p>
+
+    <p>However, if you experience any issues with your already running Crate cluster you can provice certain information that helps us understanding your issues better and therefore improves the solution to your problems.</p>
```

Comment:
Spell/grammar corrected: However, if you experience any issues with your already running Crate cluster you can provide certain information that helps us understand your issues better, and therefore improves the solution to your problems.

I'd suggest: If you experience any issues with a Crate cluster you're already running you can submit a report that will help us better understand and solve your problem.